src/I18n/DateFormatTrait.php

```diff
@@ -34,6 +34,14 @@
     public static $defaultLocale;
 
     /**
+     * The default PHP \DateTimeZone or string representative.
+     * See http://php.net/manual/en/timezones.php.
+     *
+     * @var string|\DateTimeZone
+     */
+    public static $defaultTimezone;
```

Comment:
Done. I also realized there was `nice()` and `parseDateTime()` that also want timezone and default timezone support.